web/themes/custom/vote_gov/templates/partial/homepage-link.html.twig

```diff
@@ -1,6 +1,6 @@
 <p class="arrow-text">
     <a href="/" title="Vote.gov" data-test="back-button">
```

Comment:
This href need to be dynamic. Replace it with this `{{ '/' ~ (language != 'en' ? language) }}`